### PR TITLE
feature/GENESIS-306-default-lang-selection-english

### PR DESCRIPTION
**Pull request by ElaCzapiewska**
if user.UserProfile.languageId is not supported in the UI the default
selected value of upper right corner UI component is EN

### CODE REVIEW

File: components/MainMenu/MainMenu.react.js [hunk 1]
```diff
@@ -394,6 +395,14 @@ class MainMenu extends ConditionalRenderComponent
         return tenantsForSupplierSwitch.includes(value);
     }
 
+    getDefaultLanguage(lang) {
+        return Object.keys(supportedLanguages).indexOf(lang) > -1 ? lang : 'en';
+    }
+
+    renderLanguageOptions() {
+        return Object.keys(supportedLanguages).map((lang, id) => <option key={id} value={lang}>{supportedLanguages[lang]}</option>);
```
Comment (jaconsta) [inline components/MainMenu/MainMenu.react.js:403]: is Id a number representing the position of the language in the array? If so, it's better to use the `lang` attribute as `key`

https://reactjs.org/docs/lists-and-keys.html#keys

**Review by jaconsta — CHANGES_REQUESTED** [review]
Nice!!! I left one comment.

Nice-to-have. To reduce the file size, would be nice to have the language selection on it's own component. What do you think?

**Comment by jaconsta** [comment]
Remember to update the branch with master.

File: components/MainMenu/MainMenu.react.js (hunk 1 above)
Comment (ElaCzapiewska) [reply components/MainMenu/MainMenu.react.js:403]: ok changed, key={lang}

**Comment by ElaCzapiewska** [comment]
ok, I moved language selection on it's own component.
this branch is derived from master.

File: components/MainMenu/Menu/components/MenuSelectLanguage/MenuSelectLanguage.react.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import Types from 'prop-types';
```
Comment (jaconsta) [inline components/MainMenu/Menu/components/MenuSelectLanguage/MenuSelectLanguage.react.js:2]: Can you please use PropTypes instead, just to follow the standart way

**Review by jaconsta — CHANGES_REQUESTED** [review]
Small adjustment, please.

File: components/MainMenu/Menu/components/MenuSelectLanguage/MenuSelectLanguage.react.js (hunk 2 above)
Comment (ElaCzapiewska) [reply components/MainMenu/Menu/components/MenuSelectLanguage/MenuSelectLanguage.react.js]: ok, done

**Review by jaconsta — APPROVED** [review]
:rocket: LGTM.

**Pull request merged**